collect_app/src/main/java/org/odk/collect/android/widgets/SpinnerMultiWidget.java

```diff
@@ -117,12 +117,12 @@ public void onClick(DialogInterface dialog, int id) {
 
                                     if (first) {
                                         first = false;
-                                        selectionText.setText(context.getString(R.string.selected)
```

Comment:
This seems like a really strange way to build a comma-separated string from an array. Couldn't it be something like `context.getString(R.string.selected, TextUtils.join(", ", answer_items))`? `%1$s, %2$s` doesn't really make any sense as a resource.